datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/mirrormaker/KafkaMirrorMakerConnectorTask.java

```diff
@@ -321,7 +325,8 @@ public void run() {
         _datastreamTask.acquire(LOCK_ACQUIRE_TIMEOUT);
       } catch (DatastreamRuntimeException ex) {
         LOG.error(String.format("Failed to acquire lock for datastreamTask %s", _datastreamTask), ex);
-        _dynamicMetricsManager.createOrUpdateMeter(CLASS_NAME, NUM_LOCK_FAILS, 1);
+        _dynamicMetricsManager.createOrUpdateMeter(generateMetricsPrefix(_connectorName, CLASS_NAME), _datastreamName,
```

Comment:
Do we really need this metrics under connector level?

Reply:
Well, it would be nice to find the metric using the same regex as we use for other connector metrics (e.g. errorRate, etc). And it's a lazy create metric, so it won't exist unless we actually see task lock acquire errors. Would it be okay to leave it under the connector level?